sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -38,7 +48,7 @@ protected OperationInternalBase(ClientDiagnostics clientDiagnostics, Response ra
         /// </code>
         /// </example>
         /// </summary>
-        public Response RawResponse { get; set; }
+        public abstract Response RawResponse { get; }
```

Comment:
Will this break anything that derives from this type?

Reply:
No. It is internal shared type, and we have only `OperationInternal` and `OperationInternalOfT` derived from it.